src/firmware_new/boards/frankie_v1/io/infineon_TLE5009_E1000.c

```diff
@@ -0,0 +1,70 @@
+#include "infineon_TLE5009_E1000.h"
+
+#include <stdlib.h>
+
+struct InfineonTLE5009E1000
+{
+    float A_x;
+    float A_y;
+    float O_x;
+    float O_y;
+    float phase_offset_x_rad;
+    float phase_difference_offset_rad;
+};
+
+InfineonTLE5009E1000_t* io_infineon_TLE5009_E1000_create(
+    float x_max_value, float x_min_value, float y_max_value, float y_min_value,
+    float x_magnitude_45_degrees, float y_magnitude_45_degrees,
+    float x_magnitude_135_degrees, float y_magnitude_135_degrees)
+{
+    // This function intentionally uses variable names identical to the sensor data-sheet
+
+    // Calculate the amplitudes
+    const float A_x = (x_max_value - x_min_value) / 2;
```

Comment:
more descriptive variable names instead of `A`, `O`, and `M` would be really helpful here

Reply:
Again, I did this to match the data sheet (as we've done in many places in the code already).

If we want I can change them.